src/web-components/product-card/product-card.js

```diff
@@ -23,45 +20,30 @@ template.innerHTML = `
         }
         /* component reset */
 
-        .images-container {
-            position: relative;
-            height: 300px;
-        }
+        .card {
+            @media(min-width: 1024px) {
+                display: flex;
+                margin: 80px 120px 0 120px;
```

Comment:
Beste is om niet je spacing te doen met al de margins. vooral als het gaat om de width. Kan je beter 100% width maken, en dan een max-width er omeheen met margin auto links en rechts. dus als en container.